src/bin/pgcopydb/pgcmd.c

```diff
@@ -1022,6 +1023,7 @@ struct ArchiveItemDescMapping pgRestoreDescriptionArray[] = {
 	INSERT_MAPPING(ARCHIVE_TAG_FUNCTION, "FUNCTION"),
 	INSERT_MAPPING(ARCHIVE_TAG_INDEX_ATTACH, "INDEX ATTACH"),
 	INSERT_MAPPING(ARCHIVE_TAG_INDEX, "INDEX"),
+	INSERT_MAPPING(ARCHIVE_TAG_LANGUAGE, "LANGUAGE"),
```

Comment:
Can we add a test for this too?

Reply:
We can have ACL/COMMENT statements for multiple objects like LANGUAGE/SCHEMA/INDEX/COLLATION etc. I think we should add a test case covering all of these.

Reply:
Will create a issue for this, let's take care of this later if need be.